packages/cloud-cognitive/src/components/MultiAddSelect/_index-with-carbon.scss

```diff
@@ -0,0 +1,8 @@
+//
+// Copyright IBM Corp. 2022, 2022
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../AddSelect/index-with-carbon';
```

Comment:
Any reason why we’re doing this as opposed to 

```scss
@use './carbon-imports';
@use './multi-add-select';
```

like we do for other components, including SingleAddSelect?

Reply:
Good question, initially it was because of the fact that `MultiAddSelect` and `SingleAddSelect` both simply use the styles from `AddSelect`, but I agree with you about changing it, both for consistency and in the event these components get their own styles the pattern will already be in place.

Reply:
Heh, I thought that might be the case but then thought to ask when I noticed SingleAddSelect was following the same patterns as the others.